### DISCUSSION TITLE
Getting ETIMEDOUT error in Azure Pipelines using offline build agent

### DISCUSSION THREAD

**Discussion by ranerijosh** [zricethezav/gitleaks#688] [Q&A]
I'm using a locally-hosted build agent that doesn't have internet access to build a project hosted in ADO Repos and the GitLeaks task fails each time with the following error:

##[error]Error: connect ETIMEDOUT 140.82.114.5:443

I'm using an in-repo config file and I cannot figure out why it keeps trying to initialize an SSH connection.

Any help would be greatly appreciated.

Thank you.

**Comment by w0rmr1d3r** [comment]
Hello @ranerijosh 
We might need more info about how are you running gitleaks in order to provide any help here.

Cheers!
